h/accounts/models.py

```diff
@@ -78,6 +78,9 @@ class User(Base):
                       default=False,
                       server_default=sa.sql.expression.false())
 
+    sidebar_tutorial_dismissed = sa.Column(
+        sa.Boolean, default=False, server_default=sa.sql.expression.false())
```

Comment:
This isn't going to stop me merging, but the formatting of this attribute is completely different from all the ones above it. Consistency in formatting in cases like this can really help with readability.